docs/stdlib/array.rst

```diff
@@ -57,15 +57,15 @@ Arrays store expressions of the *same type* in an ordered list.
 Constructing arrays
 ^^^^^^^^^^^^^^^^^^^
 
-An array constructor is an expression that consists of a sequence of
+Array constructors are an expression that allow for a sequence of
 comma-separated expressions *of the same type* enclosed in square brackets.
-It produces an array value:
+This will produce an array of the following:
 
 .. eql:synopsis::
```

Comment:
Recommend removing this since the grammar is difficult to parse and is conveyed via the examples.

Just for added clarity, the full recommendation would be to have these lines:
> Arrays are constructed by placing multiple comma-separated expressions within
> square brackets. Here are a few examples:

followed by the array examples that currently start on line 70 with no other content between.